docs/backend/index.md

```diff
@@ -35,7 +35,7 @@ security
 sending-email
 subscribers
 traversal-acquisition
-users-groups
+users-groups/index
```

Comment:
There is no users-groups/index.md ... I guess this was accidentally left after reorganizing.

```suggestion
```

Reply:
I don't know why `/index` was appended. It needs to be preserved. Done in https://github.com/plone/documentation/pull/1703/commits/45aa742c2350842846853d354621bbc96b2cac6d